css/style.css

```diff
@@ -0,0 +1,3958 @@
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.btn-search {
+  padding: 2px;
+  display: block;
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  right: 20px;
+}
+
+.btn-search:hover .icon-search,
+.btn-search:active .icon-search,
+.btn-search:focus .icon-search {
+  fill: var(--state-change-color);
+}
+
+.icon-search {
+  fill: var(--accent-color);
+  vertical-align: middle;
+  -webkit-transition: fill var(--trasition-effect);
+  -o-transition: fill var(--trasition-effect);
+  transition: fill var(--trasition-effect);
+}
+
+.btn-cart {
+  text-align: center;
+  padding: 5px 10px;
+  background-color: var(--bg-color);
+  border-radius: 50%;
+  -webkit-transition: background var(--trasition-effect);
+  -o-transition: background var(--trasition-effect);
+  transition: background var(--trasition-effect);
+}
+
+.cart-icon-header {
+  fill: var(--accent-color);
+  vertical-align: text-top;
+  -webkit-transition: fill var(--trasition-effect);
+  -o-transition: fill var(--trasition-effect);
+  transition: fill var(--trasition-effect);
+}
+
+.btn-cart:hover,
+.btn-cart:active,
+.btn-cart:focus {
+  background-color: var(--accent-color);
+}
+
+.btn-cart:hover .cart-icon-header,
+.btn-cart:active .cart-icon-header,
+.btn-cart:focus .cart-icon-header {
+  fill: var(--secondary-color);
+}
+
+/**************** Main **************************/
+
+.main {
+  display: -ms-grid;
+  display: grid;
+  row-gap: 120px;
+}
+
+/************** Hero section *********************/
+
+.hero {
+  background-image: -webkit-gradient(
+    linear,
+    left top,
+    right top,
+    color-stop(44%, #f0f6fd),
+    to(#ffffff)
+  );
+  background-image: -o-linear-gradient(left, #f0f6fd 44%, #ffffff 100%);
+  background-image: linear-gradient(90deg, #f0f6fd 44%, #ffffff 100%);
+  background-color: var(--secondary-color);
+}
+
+.hero-section {
+  padding-block: 75px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-column-gap: 110px;
+  -moz-column-gap: 110px;
+  column-gap: 110px;
+}
+
+.hero-content {
+  max-width: 562px;
+}
+
+.section-header {
+  font-family: var(--font-header);
+  font-weight: var(--font-bold);
+  font-size: 60px;
+  line-height: 1.2;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 3;
+  overflow: hidden;
+}
+
+.header-line {
+  display: block;
+}
+
+.accent-header {
+  color: var(--accent-color);
+}
+
+.hero-text {
+  margin-block: 16px 36px;
+  color: var(--opacity-70);
+}
+
+.hero-text::first-letter {
+  text-transform: capitalize;
+}
+
+.btns-header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-column-gap: 24px;
+  -moz-column-gap: 24px;
+  column-gap: 24px;
+}
+
+.btn-shop {
+  max-width: 170px;
+  text-wrap: wrap;
```

Comment:
`wrap` is default value for this property

Reply:
#Fixed